setup.py

```diff
@@ -485,6 +486,7 @@ def pxd(name):
                      'sources': ['pandas/_libs/src/datetime/np_datetime.c',
                                  'pandas/_libs/src/datetime/np_datetime_strings.c',
                                  'pandas/_libs/src/period_helper.c']},
+    '_libs.tslibs.parsing': {'pyxfile': '_libs/tslibs/parsing'},
```

Comment:
I dont' see any reason that parsing should *not* depend on util, otherwise you are adding boilerplate

Reply:
It does depend on util, but specifying util here is redundant with the `common_include` that gets pinned on later.

If we changed this entry to:

`'_libs.tslibs.parsing': {'pyxfile': '_libs/tslibs/parsing', 'pxdfiles': ['_libs/src/util'], 'include': []},`

we would get build errors (or possibly import-time) because `util` requires "src/headers/stdint.h" and "src/numpy_helper.h".  And numpy_helper requires "helper.h".  I'd prefer to specify these all explicitly and not have the `common_include` tacked on, but haven't found a way to make this work.

Reply:
I'll add util.pxd back into this in the interest of getting this merged; we can figure out what is and isn't necessary in the other threads.